tests/tck-build-logic/src/main/resources/AllowedDockerImages.txt

```diff
@@ -0,0 +1,7 @@
+mysql/mysql-server:8.0
```

Comment:
sort alphabetically?

Reply:
Yes, that will make this file easier for reading. Just one question, do we need then a task that will check if list is sorted as expected since this file will be populated by the contributors (probably most of them will just put image they want at the end of the file when creating a PR)?